### ISSUE TITLE
Non-looping alarm

### ISSUE THREAD

**Issue by tuukkaanttila** [SimpleMobileTools/Simple-Clock#176]
I'd like to request an option in the individual alarm settings, which when enabled, makes the alarm sound only once (not loop until acknowledged). Use case: wake up pre-alarm that lets you know it's time to wake up soon, without being irritating.

**Comment by tibbi** [comment]
duplicate of #58

**Comment by tuukkaanttila** [comment]
Not a duplicate. I mean an option that the audio doesn't loop.

**Comment by tibbi** [comment]
aha, that isnt on the roadmap then. Just use a reminder in the calendar app, alarms are not supposed to sound only once
